packages/snap/snap.manifest.json

```diff
@@ -22,6 +22,9 @@
     "snap_getBip44Entropy": [
       {
         "coinType": 52752
+      },
+      {
+        "coinType": 60
```

Comment:
is there some way we can document where these come from // for fututur generations

Reply:
I don't think JSON allows any sort of comments, but definitely would be a good idea to leave documentation somewhere else.

Reply:
those are documented [here](https://github.com/satoshilabs/slips/blob/master/slip-0044.md), but now updated the readme with a link there now.